packages/twenty-front/src/modules/object-record/advanced-filter/components/AdvancedFilterAddFilterRuleSelect.tsx

```diff
@@ -34,6 +33,10 @@ export const AdvancedFilterAddFilterRuleSelect = ({
 
   const { upsertRecordFilter } = useUpsertRecordFilter();
 
+  const { lastChildPosition } = useChildRecordFiltersAndRecordFilterGroups({
```

Comment:
I feel the naming could be improved here: lastChildRecordFilterGroupPosition

 also I'm challenging the usage of a hook here. I feel an util would be better:
 - useRecordFilterGroup({ recordFilterGroupId }) ==> reads from the state
 - getLastChildRecordFilterGroupPosition({ recordGroup })